src/main/java/org/cryptomator/ui/keyloading/KeyLoadingModule.java

```diff
@@ -30,13 +27,9 @@ static FxmlLoaderFactory provideFxmlLoaderFactory(Map<Class<? extends FxControll
 	@KeyLoading
 	@KeyLoadingScoped
 	static KeyLoadingStrategy provideKeyLoaderProvider(@KeyLoading Vault vault, Map<String, Provider<KeyLoadingStrategy>> strategies) {
-		try {
-			String scheme = vault.getUnverifiedVaultConfig().getKeyId().getScheme();
-			var fallback = KeyLoadingStrategy.failed(new IllegalArgumentException("Unsupported key id " + scheme));
-			return strategies.getOrDefault(scheme, () -> fallback).get();
-		} catch (IOException e) {
-			return KeyLoadingStrategy.failed(e);
-		}
+		String scheme = vault.getUnverifiedVaultConfig().getKeyId().getScheme();
+		var fallback = KeyLoadingStrategy.failed(new IllegalArgumentException("Unsupported key id " + scheme));
+		return strategies.getOrDefault(scheme, () -> fallback).get();
```

Comment:
Please undo this change. This relies heavily on the UI to not allow any unlock unless a vault config is present. Which is not very resilient.

Reply:
Fixed in 76811f9